latest/linux/amd64/TorchInstaller.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
consider merging these two TorchInstaller scripts into a single one. also double check with @alankbi that all this stuff is required since I recall us discussing that it may not be the case.